lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -964,10 +970,10 @@ public static DataSize writeOrcColumnHive(File outputFile, Format format, Compre
             throws Exception
     {
         RecordWriter recordWriter = createOrcRecordWriter(outputFile, format, compression, type);
-        return writeOrcFileColumnHive(outputFile, recordWriter, type, values);
+        return writeOrcColumnHive(outputFile, recordWriter, type, values);
     }
 
-    public static DataSize writeOrcFileColumnHive(File outputFile, RecordWriter recordWriter, Type type, Iterator<?> values)
+    public static DataSize writeOrcColumnHive(File outputFile, RecordWriter recordWriter, Type type, Iterator<?> values)
             throws Exception
     {
         SettableStructObjectInspector objectInspector = createSettableStructObjectInspector("test", type);
```

Comment:
why don't we reuse the new method you just added?

Reply:
due to it's actually similar with `writeOrcHiveSingleStringColumn` now but due to differs in `createOrcRecordWriter(File outputFile, Format format, CompressionKind compression, Type type)` with using `orc.bloom.filter` does not the compatible. for that it's need separate additional refactoring. to do that?

Reply:
```
        SettableStructObjectInspector objectInspector = createSettableStructObjectInspector("test", type);

        writeOrcColumnsHive(
                recordWriter,
                objectInspector,
                ImmutableList.of(type),
                StreamSupport.stream(
                                Spliterators.spliteratorUnknownSize(values, Spliterator.ORDERED),
                                false)
                        .map(value -> (Function<Integer, Object>) index -> value).iterator());
```

I used in `io.trino.orc.OrcTester#writeOrcColumnsHive(org.apache.hadoop.hive.ql.exec.FileSinkOperator.RecordWriter, org.apache.hadoop.hive.serde2.objectinspector.SettableStructObjectInspector, java.util.List<io.trino.spi.type.Type>, java.util.Iterator<java.util.function.Function<java.lang.Integer,java.lang.Object>>)` 
 `SettableStructObjectInspector` instead of `StandardStructObjectInspector`

Reply:
If this makes sense, please do remove as well `io.trino.orc.OrcTester#writeOrcHiveSingleStringColumn` method because it is not necessary anymore.

Reply:
make sense. in progress.

Reply:
done